package.json

```diff
@@ -19,7 +19,8 @@
     "test": "node_modules/.bin/jest",
     "start": "node server.js",
     "build": "node_modules/.bin/gulp build",
-    "postinstall": "npm run build"
+    "postinstall": "npm run build",
+    "deploy_assetss": "bin/deploy_assetss"
```

Comment:
Should the script name and filename be `deploy_assets`, or is there a reason for extra S in `assetss`?  Does your keyboard have a sticky `S` key?